CMakeLists.txt

```diff
@@ -53,11 +55,12 @@ ADD_SUBDIRECTORY(tests)
 ENABLE_TESTING()
 ADD_TEST(test_pfx tests/test_pfx)
 ADD_TEST(test_lpfst tests/test_lpfst)
-ADD_TEST(test_pfx_locks tests/test_pfx_locks)
+#ADD_TEST(test_pfx_locks tests/test_pfx_locks)
```

Comment:
It takes a lot of time to complete, I ran ```test_pfx_locks``` on my notebook and it took 92 minutes to complete. IMHO such a long, intensive does not qualify to be included in Travis CI testing!